README.md

```diff
@@ -276,12 +276,12 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 > 
 >     1     
  
-**PC_Envelope(p pcpatch)** returns **bytea**
+**PC_Envelope_AsBinary(p pcpatch)** returns **bytea**
```

Comment:
Ok, I get your point. I don't think it's very useful but we can keep it. I'd just suggest to rename it to `PC_EnvelopeAsBinary` for consistency reasons. Neither PointCloud nor PostGIS uses a mix of CamelCase and Underscore notations.